### PR TITLE
Update rooms-operations.md

### PR DESCRIPTION
**Pull request by flinchbot**
Added references to Teams Admin Center in remote support. Removed "skype" from "Skype account config" as it is legacy and potentially confusing. You can also do all these changes using skypesettings.xml. I've not added that as we want to focus attention towards admin center and away from xml files.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/rooms/rooms-operations.md [hunk 1]
```diff
@@ -58,11 +58,11 @@ The following table summarizes the possible remote operations and the methods yo
 
 |Workgroup|Not domain joined|Domain joined|
 |:-----|:-----|:-----|
-|Restart  <br/> |Remote desktop  <br/> Remote Powershell  <br/> |Remote desktop (requires further configuration)  <br/> Remote Powershell (requires further configuration)  <br/> Configuration Manager  <br/> |
+|Restart  <br/> |Teams Admin Center  <br/> Remote desktop  <br/> Remote Powershell  <br/> | <br/>Remote desktop (requires further configuration)  <br/> Remote Powershell (requires further configuration)  <br/> Configuration Manager  <br/> |
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-operations.md:61]:
```suggestion
|Restart  <br/> |Teams admin center  <br/> Remote desktop  <br/> Remote Powershell  <br/> | <br/>Remote desktop (requires further configuration)  <br/> Remote Powershell (requires further configuration)  <br/> Configuration Manager  <br/> |
```

File: Teams/rooms/rooms-operations.md [hunk 2]
```diff
@@ -58,11 +58,11 @@ The following table summarizes the possible remote operations and the methods yo
...
 |Update OS  <br/> |Windows Update  <br/> |Windows Update  <br/> WSUS  <br/> |
 |App update  <br/> |Windows Store  <br/> |Windows Store  <br/> Configuration Manager  <br/> |
-|Skype Account Config  <br/> |Not currently supported  <br/> |Not currently supported  <br/> |
-|Access logs  <br/> |Not currently supported  <br/> |Not currently supported  <br/> |
+|Account Config  <br/> |Teams Admin Center  <br/> |Teams Admin Center  <br/> |
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-operations.md:64]:
```suggestion
|Account Config  <br/> |Teams admin center  <br/> |Teams admin center  <br/> |
```

File: Teams/rooms/rooms-operations.md [hunk 3]
```diff
@@ -58,11 +58,11 @@ The following table summarizes the possible remote operations and the methods yo
...
+|Access logs  <br/> |Teams Admin Center  <br/> |Teams Admin Center <br/> |
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-operations.md:65]:
```suggestion
|Access logs  <br/> |Teams admin center  <br/> |Teams admin center <br/> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**